torch/distributed/_spmd/distribute.py

```diff
@@ -129,10 +129,28 @@ def _update_specs_for_redistribute(args, target_schema, redistribute):
     return specs, unflattened_args
 
 
+# When no tensor redistribution is required, we only need to update non-tensor args
+# of the node according to op_schema and avoid building a GraphModule just for the
+# node.
+def _update_node_from_op_schema(node: torch.fx.Node, op_schema: OpSchema) -> None:
```

Comment:
TODO for @mrshenli: move this utility from SPMD to DTensor, as dispatch decision is made by DTensor.